docs/element-templates/README.md

```diff
@@ -120,7 +120,19 @@ As seen in the code snippet a template consist of a number of important componen
 * `appliesTo : Array<String>`: List of BPMN types the template can be applied to.
 * `properties : Array<Object>`: List of properties of the template.
 
+### JSON Schema Compatibility
 
+The application uses the `$schema` property to ensure compatibility for a given element template. The current supported [Camunda Element Templates JSON Schema version](https://github.com/camunda/element-templates-json-schema) is `v0.3`. Element Templates defining a `$schema` version higher than supported will be ignored. 
+
+For example, given the following `$schema` definition, the application takes `0.2.0` as the JSON Schema version of the element template.
+
+```json
+"$schema": "https://unpkg.com/@camunda/element-templates-json-schema@0.2.0/resources/schema.json"
+```
+
+The JSON Schema versioning is backward-compatible, meaning that all versions below are still accepted. In case no `$schema` is defined, we will assume the latest JSON Schema version.
```

Comment:
```suggestion
The JSON Schema versioning is backwards-compatible, meaning that all versions including or below the current one are supported. In case no `$schema` is defined, the Camunda Modeler assumes the latest JSON Schema version.
```

Reply:
Grammarly says "backward-compatible" is correct ✏️